services/account.js

```diff
@@ -50,14 +50,69 @@ class AccountService {
     }
   }
 
+  async withdraw({ amount, number, atm_id }) {
+    const t = await sequelize.transaction();
+    const [findAccount, findAtm] = await Promise.all([
+      Account.findOne({
+        where: { number },
+        attributes: ['balance'],
+      }),
+      Atm.findOne({
+        where: { id: atm_id },
+        attributes: ['balance'],
+      }),
+    ]);
+    if (amount > 0) {
+      if (amount <= findAtm.balance) {
```

Comment:
tem que ser < e não <= se for igual ainda pode fazer o saque porem o saldo fica como 0

Reply:
foi proposital pq se o cliente tiver 1 real ele tem direito a sacar

Reply:
altero?